dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/relationship/DefaultRelationshipService.java

```diff
@@ -90,6 +91,16 @@ public Relationship getRelationship(String uid) throws ForbiddenException, NotFo
     return map(relationship);
   }
 
+  @Override
+  public List<Relationship> getRelationships(@Nonnull List<String> uids)
```

Comment:
Will we use our UID type in a separate PR?

Should we create a task for fetching relationships for uids all at once instead of one at a time?

Reply:
Yes and yes.
I think when I manage to remove `RelationshipService` and `RelationshipStore` from the core, then I will review our tracker service and store.